src/cgw_connection_processor.rs

```diff
@@ -290,8 +292,19 @@ impl CGWConnectionProcessor {
                             }
                             if self.feature_topomap_enabled {
                                 let topo_map = CGWUCentralTopologyMap::get_ref();
-                                topo_map.process_state_message(&device_type, evt).await;
-                                topo_map.debug_dump_map().await;
+
+                                // TODO: remove this Arc clone:
```

Comment:
Is this comment still relevant or should be removed?

Reply:
Not really. it states that #306 should be removed, and some other way of accessing NB API should be made further on;
Currently, a copy of Arc<Server> is being passed around and copied, and we potentially have to come up with a new approach;